tasks/cli.yaml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: read ansible_min_version from meta data
```

Comment:
Should this go into a dedicated `tasks/validate.yaml` which all other `tasks` include?

Reply:
@gundalow see https://github.com/ansible-network/network-engine/pull/131. You can add this validation in main.yaml instead as it is entrypoint

Reply:
I have a slightly different pattern that makes this work a little more seamlessly within the roles.  Take a look at the following two links:

https://github.com/ansible-network/arista_eos/blob/devel/tasks/load_config.yaml#L3
https://github.com/ansible-network/arista_eos/blob/devel/includes/init.yaml

The above pattern creates an init set of tasks where we can centrally handle all role validation instead of having to manage individual ones and make sure they are propagated to every role, every time.

Reply:
+1